packages/oruga-next/src/composables/defineClasses.ts

```diff
@@ -23,7 +23,7 @@ import type {
 } from "@/types";
 
 // named tuple as prop definition
-type ComputedClass = readonly [
+export type ComputedClass = readonly [
```

Comment:
I think the export is no longer needed, isn't it?

Reply:
In the end I didn't need it, but I think it's still a good idea to export it so it can be used in other files. That type is used implicitly all over the place and it can be handy for debugging at the very least. At worst it's a few extra bytes in the package which will never make it into anyone's production build.

Reply:
But it will never be needed outside, because we define all defineClasses props directly at the function call. Just a programming paradigm like don't add code that is not used.

Reply:
fair enough, removed.